### PR TITLE
Add 409 status for errors when creating a user

### PR DESCRIPTION
**Pull request by mwitte98**
Related Issue #51

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def create
       }
       render json: { 'logged_in': true }
     else
-      render json: { errors: user.errors.full_messages }
+      render json: { errors: user.errors.full_messages }, status: :conflict
```
Comment (7imbrook) [inline app/controllers/users_controller.rb]: Can we actually check if there's a conflict for this? And not just assume its in the email

**Review by 7imbrook — CHANGES_REQUESTED** [review]

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (mwitte98) [reply app/controllers/users_controller.rb]: Do you want a 409 http status for email conflicts and a 400 http status for all other errors?

Comment (7imbrook) [reply app/controllers/users_controller.rb]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

Comment (mwitte98) [reply app/controllers/users_controller.rb]: #55 will have to be completed before I can continue with this. Any changes will break the build due to rubocop

**Comment by mwitte98** [comment]
This will break the build but once #59 is merged with it, it'll be fixed

**Review by 7imbrook — APPROVED** [review]

**Pull request merged**